mne/time_frequency/tests/test_tfr.py

```diff
@@ -430,6 +430,47 @@ def test_plot():
     plt.close('all')
 
 
+def test_plot_joint():
```

Comment:
Considering how much of a speed boost #4225 and #4224 are giving us, can you pass `{"res": 8, "contours": 0, "sensors": False}` to as many of the underlying topomap calls as possible?